domino/routes.py

```diff
@@ -151,3 +151,7 @@ def hardware_tiers_list(self, project_id):
     def find_project_by_owner_name_and_project_name_url(self):
         return f'{self.host}/v4/gateway/projects/findProjectByOwnerAndName' \
                f'?ownerName={self._owner_username}&projectName={self._project_name}'
+
+    # User URLs
+    def users_get(self):
+        return self.host + f'/v4/users'
```

Comment:
This is beyond the scope of this PR obviously, but it makes me sad that we don't have an API to return a single user from the DB, but instead have to retrieve all users on the system, then search though them in application code.